SonarQube.TeamBuild.PreProcessor/TeamBuildPreProcessor.cs

```diff
@@ -108,12 +108,7 @@ public bool Execute(ProcessedArgs args, ILogger logger)
 
         private bool FetchArgumentsAndRulesets(ProcessedArgs args, AnalysisConfig config, ILogger logger)
         {
-            string hostUrl = GetServer(args);
-            if (String.IsNullOrEmpty(hostUrl))
-            {
-                logger.LogError(Resources.ERROR_NoHostUrl);
-                return false;
-            }
+            string hostUrl = args.GetSetting(SonarProperties.HostUrl);
```

Comment:
We are expecting the host url to have been validated before we get to this point (GetSetting will throw if the url isn't set)